Kconfig

```diff
@@ -270,4 +270,10 @@ config PERFORMANCE_COUNTERS
 	  use the stamp() macro periodically to find out how long the cpu
 	  was in active/sleep state between the calls and estimate the cpu load.
 
+config LPS_COUNTERS
+	bool "LPS counters"
+	default n
+	help
+	  Select to enable simple debug LPS counters
```

Comment:
Can you add a little more context about the LPS counters here. e.g. what they are used for and how they can be used.

Reply:
This probably should not be named "LPS" which is Intel platform specific mechanism if the counters are going to occupy fw regs space which is a common generic interface. So some common abstract D0, D1, D2 state definitiion could be introduced with three 64-bit counters in the fw regs space to measure residency in each one. Intel platforms could map HPRO to D0, LPRO to D1, and LPRO+LPS to D2. State transitions could be captured inside the pm_runtime() (probably clk api as well) as deltas between the previous and current value of the platform timer (art), and deltas accumulated inside the fw regs space. Then a counter/sum would give % residency.

Reply:
@mmaka1 How do you envision adding further granularity to the counters ? By adding next states (D4, D5) or by adding suffixes (D1.1..D3.3) to existing counters ? If D1..D3 concept was based on PCI Express spec idea to make abstraction HW implementation agnostic then for D3 we have D3hot and D3cold states. I'm OK with D1..D3 if we have agreement to add substrates in the future as needed.

Reply:
@lbetlej D as a symbol seems to confusing, PCI device state uses it as you mentioned. Since it represents just DSP core residency, let's replace with R as discussed. @abonislawski it seems to be the last thing to change, technically all recent updates are OK, imo RFC tag may be removed from the subject.

Reply:
@lbetlej @mmaka1 updated with R states